client/src/components/App/Carousel.jsx

```diff
@@ -1,40 +1,46 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const StyledCarousel = styled.div`
+  display: flex;
+  flex-direction: ${props => props.direction};
+`;
 
 /**
  * Displays a set of items with buttons to scroll through them
  * @param {Array}  items The items to be displayed
  * @param {Number} size  How many items to display at once
  */
-function Carousel({ items, size }) {
+function Carousel({ items, size, direction = 'row' }) {
   const [start, setStart] = useState(0);
-
   const decrement = (event) => {
     event.preventDefault();
     setStart((oldStart) => Math.max(0, oldStart - size));
   };
 
   const increment = (event) => {
     event.preventDefault();
-    setStart((oldStart) => Math.min(items.length - size + 1, oldStart + size));
+    setStart((oldStart) => Math.min(items.length - size, oldStart + size));
```

Comment:
I believe you have this to prevent it from going over, not sure exactly how it works but if it's not working you could also just test if the value is the max before and if it is change it, it's more code, and if this is working it seems better

Reply:
Yeah that's to prevent going over the end range. I think I'm going to refactor to make increment move by one, disable the button at end range, and call increment `size` number of times.